external/source/exploits/CVE-2022-26904/SuperProfileDLL/.vs/SuperProfileDLL/project-colors.json

```diff
@@ -0,0 +1,11 @@
+{

```

Comment:
Pretty sure this file can be removed from version control.

Reply:
Removed .vs directory with https://github.com/rapid7/metasploit-framework/pull/16382/commits/bab215fccb4172a0a4e571b5a8ae66fc7b505341